src/views/AssetsList.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <BaseContentBlock :title="$t('assets')" class="assets-list-page">
+    <BaseTable
+      :loading="table.loading.value"
+      :pagination="table.pagination"
+      :items="table.items.value"
+      @next-page="table.nextPage()"
+      @prev-page="table.prevPage()"
+      @set-page="table.setPage($event)"
+      @set-size="table.setSize($event)"
+    >
```

Comment:
You can refactor `BaseTable` to accept just what `useTable()` returns so you don't need to configure all these props:

```html
<BaseTable :table="table" />
```

Reply:
I left the possibility for custom logic for tables. but probably in this project such freedom is not required. I will redo it in the next pr